infra/docker-compose.development.yml

```diff
@@ -19,5 +19,6 @@ services:
     ports:
       - '${EMAIL_SMTP_PORT}:1025'
       - '${EMAIL_HTTP_PORT}:1080'
+
```

Comment:
Quando for fazer os próximos ajustes, pode desfazer essa modificação para o arquivo não aparecer na lista.